cluster/charts/rook-ceph-cluster/templates/ingress.yaml

```diff
@@ -1,6 +1,8 @@
 {{- if .Values.ingress.dashboard.host }}
 ---
-{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
+apiVersion: networking.k8s.io/v1
+{{ else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
 apiVersion: networking.k8s.io/v1beta1
 {{ else }}
 apiVersion: extensions/v1beta1
```

Comment:
Is this condition still needed?

Reply:
Depends on which k8s versions rook supports. `networking.k8s.io/v1` was introduced in 1.19 and `networking.k8s.io/v1beta1` was introduced in 1.14. Do we want to keep the latter to support <1.19 or remove it and force those users install an older version of the chart?

Reply:
I meant this one `apiVersion: extensions/v1beta1`

Reply:
Ah, yeah. That one's pretty old but the docs say [1.11 is the minimum supported version](https://rook.io/docs/rook/v1.7/k8s-pre-reqs.html#minimum-version) so it should probably stay.